### ISSUE TITLE
LC0068 - GetBySystemId

### ISSUE THREAD

**Issue by ans-bar-bm** [StefanMaron/BusinessCentral.LinterCop#1215]
It seems the usage of `GetBySystemId` does not trigger a check for read permissions on the table for the LC0068 rule.
```AL
procedure Test(MyGuid: Guid)
var
    Customer: Record Customer;
begin
    Customer.GetBySystemId(MyGuid); // no LC0068 reported for read permissions
end;
```

**Comment by Arthurvdv** [comment]
Hi @ans-bar-bm,

Thanks for reporting this and indeed `GetBySystemId` is not being detected as a read operation for LC0068.

This now has been resolved as part of ALCops/Analyzers#224 and will ship with the upcoming release of ALCops v0.8.0.

I'm closing this issue here since, as noted in the README, this repository is deprecated and development continues at ALCops.

If you'd like to migrate, the documentation at alcops.dev is a good starting point, and there's a dedicated migration guide at alcops.dev/docs/lintercop-migration (https://alcops.dev/docs/lintercop-migration/).

Feel free to open issues or contribute directly in the ALCops/Analyzers (https://github.com/ALCops/Analyzers) repository going forward.
